### DISCUSSION TITLE
技巧分享：沙拉划词配合youtube字幕划词

### DISCUSSION THREAD

**Discussion by taikulawo** [crimx/ext-saladict#802] [Show and tell]
#### 问题：

Youtube生成的字幕默认情况下不能选择，且每次点击字幕时，字幕会突然向上滚动

![you](https://user-images.githubusercontent.com/24750337/82004880-80e56700-9696-11ea-9a2b-7fca6830b2b6.gif)
### 解决
为了配合沙拉划词，我写了一个油猴脚本，将youtube字幕改为可选择，并且直接固定死字幕，点击时字幕不会突然向上滑动，避免了双击字幕时字幕滑动之后全屏了播放器

https://greasyfork.org/en/scripts/402598-youtube-entancer-of-mine/code

现在我将划词面板固定到右角，碰到字幕单词直接点击一下，面板就会自动翻译，十分方便

![image](https://user-images.githubusercontent.com/24750337/82004593-e127d900-9695-11ea-88a2-4d0c13996adc.png)

感谢插件作者提供的生产力工具，同时感谢大家分享的各种技巧，以及对我这个程序员英语的帮助，这是我的一丁点贡献，感谢😁

**如果脚本有问题，直接在这个Issues下@我，greasyfork我很少用**

**Comment by crimx** [comment]
这看起来很赞👍感谢分享！

![1601086345410](https://user-images.githubusercontent.com/6882794/94327912-0122be00-ffe1-11ea-982a-6e1f628146b7.jpg)

**Reply by lulumengyi** [reply]
太棒了！谢谢！

**Comment by james-xiao** [comment]
非常感谢！

**Comment by taikulawo** [comment]
@james-xiao 是你在https://greasyfork.org 提的反馈吗，我修复了，你可以再试试

**Comment by emily22333** [comment]
感谢分享

**Comment by middzwb** [comment]
感谢！

**Comment by BasaNite16** [comment]
@iamwwc 你好作者，今天使用了你的这个脚本（0.9.1），发现依然无法固定captions。不知道是否失效了还是我配置问题？

**Reply by taikulawo** [reply]
> @iamwwc 你好作者，今天使用了你的这个脚本（0.9.1），发现依然无法固定captions。不知道是否失效了还是我配置问题？

我特地试了下，没问题，录个屏看下？

**Reply by BasaNite16** [reply]
感谢回复，已解决。检查过发现是和另一个YT插件产生冲突了。

**Comment by xiajunwen1007** [comment]
@iamwwc  不知道能不能实现鼠标放在字幕上时自动暂停，移走后再自动恢复播放。现在这样比较适合查词量比较小的情况，当要查的数量变多时，不太方便。非常谢谢作者的脚本！
